zulipterminal/model.py

```diff
@@ -1253,6 +1253,53 @@ def _get_all_stream_ids(self) -> List[int]:
         id_list.extend(stream_id for stream_id in self._never_subscribed_streams)
         return id_list
 
+    STREAM_KEYS = Literal[
+        "stream_id",
+        "name",
+        "description",
+        "rendered_description",
+        "date_created",
+        "invite_only",
+        "subscribers",
+        "is_announcement_only",
+        "stream_post_policy",
+        "is_web_public",
+        "message_retention_days",
+        "history_public_to_subscribers",
+        "first_message_id",
+        "stream_weekly_traffic",
+    ]
+    SUBSCRIPTION_KEYS = Literal[
+        STREAM_KEYS,
+        "desktop_notifications",
+        "email_notifications",
+        "wildcard_mentions_notify",
+        "push_notifications",
+        "audible_notifications",
+        "pin_to_top",
+        "email_address",
+        "is_muted",
+        "role",
+        "color,",
+    ]
+
+    def stream_property(
+        self, stream_id: int, property: STREAM_KEYS
+    ) -> Optional[Union[int, str, bool, List[int]]]:
+        return self._get_stream_from_id(stream_id)[property]
```

Comment:
How well does this approach work in practice? Does this lead to typing errors, since somehow one has to map the result types to the property names?